Riot/Categories/MXRoom.swift

```diff
@@ -0,0 +1,35 @@
+// 
+// Copyright 2025 New Vector Ltd
+//
+// SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+// Please see LICENSE files in the repository root for full details.
+//
+
+@objc
+extension MXRoom {
+    /// Returns true if the user is the last owner of the room, but not the last member.
```

Comment:
From what I can see below with the `dispatch_queue_main`/`MainActor.run`, I think it might be worth adding a warning in the doc comment that there's no guarantee this method returns on the same thread.